files/plugins/rabbitmq_status.py

```diff
@@ -172,6 +174,29 @@ def _get_queue_metrics(session, metrics, host, port):
     }
 
 
+def _get_consumer_metrics(session, metrics, host, port):
+    VHOSTS = []
+    vhosts_response = _get_rabbit_json(session, VHOSTS_URL % (host, port))
+    # get the vhosts
+    for vhost in vhosts_response:
+        VHOSTS.append(vhost['name'])
+
+    queues_without_consumers = 0
+    for vhost in VHOSTS:
+        # uri encode /
+        URI_ENCODED = vhost.replace('/', '%2F')
+        queue_response = _get_rabbit_json(session, CONSUMERS_QUEUES_URL %
+                                          (host, port, URI_ENCODED))
+    for queue in queue_response:
```

Comment:
By the looks of it this for-loop needs indenting otherwise `queue_response` will only ever relate to the last value of `vhost`.